src/ui/resource/Image.js

```diff
@@ -28,13 +28,19 @@ import device;
 import lib.PubSub;
 import event.Callback as Callback;
 import ui.resource.loader as resourceLoader;
+import cache.LRUCache as LRUCache;
 
+var TINT_CACHE_SIZE = 4096;
```

Comment:
Agreed, I figured this would probably need to be run through in different test cases. I put it at this because PIXI's caching limitations were enormous, and this is a small percentage of what they potentially allow. This stuff only comes into play when tinting a lot of objects.